drivers/soundwire/intel_auxdevice.c

```diff
@@ -44,6 +47,39 @@ static int md_flags;
 module_param_named(sdw_md_flags, md_flags, int, 0444);
 MODULE_PARM_DESC(sdw_md_flags, "SoundWire Intel Master device flags (0x0 all off)");
 
+struct wake_capable_part {
+	const int mfg_id;
+	const int part_id;
+};
+
```

Comment:
we can align the definition of mfg_id and part_id with slave definition, since we compare them

````c
struct sdw_slave_id {
	__u16 mfg_id;
	__u16 part_id;
	__u8 class_id;
	__u8 unique_id;
	__u8 sdw_version:4;
;
````

Reply:
I thought about it but we are not going to use the unique_id or the class_id, and then the sdw_version field is also useless - at least for now. In the end I ended up creating a separate/simpler definition with just what we care about.

Reply:
sorry,  I meant using __u16, not int.  comparing u16 to int will cause warning.

Reply:
yes that's right, thanks for the suggestion @RanderWang

Reply:
done